app/controllers/charges_controller.rb

```diff
@@ -113,7 +113,11 @@ def validate(seller_id:, line_items:)
   def create_square_payment_request(
     nonce:, amount:, email:, name:, seller:, line_items:, is_distribution:
   )
-    square_location_id = seller.square_location_id
+    square_location_id = if is_distribution && !seller.non_profit_location_id.blank?
```

Comment:
is !.blank? the same as .present?

Reply:
ahah realized that it is! switched to .present? as I think the positive case makes more sense.